ReactWindows/ReactNative.Net46/UIManager/UIManagerModule.cs

```diff
@@ -113,8 +120,7 @@ public int AddMeasuredRootView(SizeMonitoringCanvas rootView)
             var width = rootView.ActualWidth;
             var height = rootView.ActualHeight;
 
-            var context = new ThemedReactContext(Context);
```

Comment:
The ThemedReactContext wrapped the "ReactContext" in a peculiar way, which required that a ReactInstance be available.  This limited the ability to test UIManagerModule and ViewManagers without having to build a ReactInstance.   It turns out that the ThemedReactContext was being passed through the various layers, but was not being used for anything other than getting access to the UIManagerModule's EventDispatcher via a call to GetNativeModules...